### DISCUSSION TITLE
Add Gaussian Splatting Ability

### DISCUSSION THREAD

**Discussion by newguy123-creator** [alicevision/Meshroom#2462] [Ideas]
It would be great, if we could get the ability to do Gaussian Splatting with Meshroom, and get a .PLY or .SPLAT file out of it.
Not sure if this is only feasible on Windows and Linux systems, as I'm unsure if CUDA is a requirement or not.

**Comment by natowi** [comment]
https://github.com/alicevision/MeshroomResearch
```
Optimization-based via NerfStudio (Upcoming):

    Instant-ngp
    3D Gaussian Splatting
```

**Reply by newguy123-creator** [reply]
So I would need Nerfstudio installed and it would process stuff in there?
I was hoping to do everything inside Meshroom without the need to install Nerfstudio, otherwise I could just do everything directly in Nerfstudio from the start. 

Therefore my request to do things in Meshroom, so that I dont need Nerfstudio and its million dependencies installed

**Reply by natowi** [reply]
I totally get it. In the MeshroomResearch Repository you can see things, that are being worked on but are not yet ready to be included in the main release. We need to avoid to blow up the installer size of Meshroom and handle all the dependencies.

Currently a plugin system is in the works to make it easier for other developers to extend Meshroom.

**Reply by newguy123-creator** [reply]
ok thanks, looking forward to it
